src/general/SceneDisplayer.cs

```diff
@@ -23,7 +23,7 @@ public string Scene
 
     private void LoadNewScene()
     {
-        if (currentlyShown != null)
+        if (currentlyShown != null && IsInstanceValid(currentlyShown))
```

Comment:
I'll just make an observation regarding this as I don't have time to investigate now: so this now asks Godot "hey have we accidentally deleted this currentlyShown scene"? I think a better approach would be investigating where currentlyShown is is freed but not set to null. 

Would adding a line after 29 like `currentlyShown = nul;l`, fix this?

Reply:
Yes and Yes.

Reply:
Is cytoplasm.tscn supposed to be loaded in organel.Definition.Display Scene??Cause its not.

Reply:
It's not because the DisplayScene is what's shown in the microbe for that organelle. Cytoplasm only has a scene for when it is used as a corpse chunk.

Reply:
Im saying because that also seems to fix the problem.

Reply:
I suppose that also *masks* the problem. I want to fix the actual problem which is that `currentlyShown` is sometimes freed without setting it to null. That's the change that actually addresses the underlying problem, and one I want added to the codebase.

Reply:
Godot doesnt set it to null.it marks it as unused.Doesnt remove all references to the removed object.thats why the null check doesnt work.

Reply:
That's why the correct thing is in our code to set the variable to null after deleting that. That solves all the problems and is valid usage. It's completely nonsensical to me to not do things properly here and instead rely on asking the engine if we did something stupid with an object. I have experience in C++ and to me this seems as silly as deleting a pointer in C++ without setting it to null, the difference being there that you can't ask the runtime "did I delete this already?", instead you need to handle it yourself.

Reply:
"I have experience in C++ and to me this seems as silly as deleting a pointer in C++ without setting it to null"
Godot has its own type of null.And that is "unused."And aparently null here means uninstantaiated and "unused" is the actuall "null".And thats the check.I dont think were doing any kind of stupid things??
So?How do you wnat it boss?Set to null?

Reply:
Set the C# variable to null. There's a null check on this *exact* line which will prevent the problem of freeing a Godot object multiple times.